src/components/navigation/CSidebar.vue

```diff
@@ -214,8 +226,31 @@ export default {
     },
 
     pin () {
+      this.saveSettings(!this.isPinned)
       this.isPinned = !this.isPinned
-      // this.isExpanded = !this.isExpanded
+    },
+
+    defaultSidebarAppearance () {
+      const sidebar = JSON.parse(window.localStorage.getItem('sidebar_settings'))
+      if (sidebar) {
+        this.isExpanded = sidebar.expanded
+        this.isPinned = sidebar.pinned
+      } else {
+        if (window.innerWidth < 500) {
+          this.closeSidebar()
+        } else {
+          this.openSidebar()
```

Comment:
Why open sidebar here ?

Reply:
Sidebar should open by default if resoultion bigger than 500 and prevoiusly no record in local storage

Reply:
Hmm i think problems could occur if this happens. For example on compose on namespace list, the sidebar is not expanded. Same on workflows and reporter. So if you open it by default it would break that (probably). You would need to test that extensively. 
And the same happens with local storage. So it could open when its on the wrong page.